### PR TITLE
Fixed bugs in Dev branch

### PR DESCRIPTION
**Pull request by techdecipher**
This bug fix is related with header of techdecipher.com

### CODE REVIEW

**Comment by techdecipher** [comment]
Hello @Waytopranav  please review the  PR

File: dump.html [hunk 1]
```diff
@@ -1 +1,3 @@
 This is test file
+
```
Comment (Waytopranav) [inline dump.html:2]: Looks good yeah

**Review by Waytopranav — APPROVED** [review]
The changes look good

**Comment by techdecipher** [comment]
Thank you

**Pull request merged**